LoRaEngine/LoraKeysManagerFacade/IDeviceRegistryManager.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade
```

Comment:
Enable nullable context in new source files:

```suggestion
#nullable enable

namespace LoraKeysManagerFacade
```